src/applications/personalization/profile/tests/components/account-security/AccountSecurityContent.unit.spec.jsx

```diff
@@ -115,18 +115,23 @@ describe('AccountSecurityContent', () => {
     wrapper = shallow(<AccountSecurityContent {...makeDefaultProps()} />);
     const alertBox = wrapper.childAt(1);
     const alertBoxText = alertBox.find('p');
-    const alertBoxLink = alertBox.find('a');
+    const firstAlertBoxLink = alertBox.find('a').first();
```

Comment:
not saying you should change this, but I have really grown to hate enzyme tests like this! 😆

Reply:
Yeah, they are pretty ugly! 💀